src/app/api/health/healthCheck.utils.ts

```diff
@@ -1,38 +1,15 @@
-import { BlockChangeLog } from '@/app/api/utils/db.schema'
-import { config } from '@/config'
-import { STATE_SYNC_BLOCK_STALENESS_THRESHOLD } from '@/lib/constants'
-import { db } from '@/lib/db'
-import { getBlockNumber } from 'wagmi/actions'
-import { BlockNumberFetchError } from './healthCheck.errors'
+import strategies from './strategies'
 
 const HEALTH_CHECK_TYPE = ['lastBlockNumber'] as const
 
 type HealthCheckType = (typeof HEALTH_CHECK_TYPE)[number] | 'all'
 
 type HealthCheckResult = boolean
 
-const _lastBlockNumber = async (): Promise<boolean> => {
-  const blockChangeLog = await db<BlockChangeLog>('BlockChangeLog').orderBy('blockTimestamp', 'desc').first()
-
-  if (!blockChangeLog) {
-    return false
-  }
-
-  const blockNumberOnChain = await getBlockNumber(config)
-
-  if (!blockNumberOnChain) {
-    throw new BlockNumberFetchError()
-  }
-
-  return (
-    BigInt(blockChangeLog.blockNumber) + BigInt(STATE_SYNC_BLOCK_STALENESS_THRESHOLD) >= blockNumberOnChain
-  )
-}
-
-type HealthCheckFunction = () => Promise<boolean>
+type HealthCheckFunction = (...args: any[]) => Promise<boolean> | boolean
```

Comment:
```suggestion
type HealthCheckFunction = (...args: unknown[]) => Promise<boolean> | boolean
```
Should we use unknown instead of any?

Reply:
done